### ISSUE TITLE
One question in dis_update function

### ISSUE THREAD

**Issue by laulampaul** [Wenchao-Du/LIR-for-Unsupervised-IR#6]
in dis_update(self, x_a, x_b, hyperparameters):
the content adversial codes are ,
       out_a = self.dis_content(h_a)
      out_b = self.dis_content(h_b)
why not use h_a.detach() and h_b.detach() to avoid the update in the parameters  of G ?
Thank you!
